braindecode/models/functions.py

```diff
@@ -109,3 +112,319 @@ def drop_path(x,
     if keep_prob > 0.0 and scale_by_keep:
         random_tensor.div_(keep_prob)
     return x * random_tensor
+
+
+
+
+
+def resample(
```

Comment:
I don't think this code should be in braindecode.models maybe we need a braindecode.functional ?

Reply:
Yes! I have a similar thing for the augmentation. I am open to doing this in another PR because it will involve more models. Issue opened #585.